### PR TITLE
FUT-34 seasonal summary feature

### PR DESCRIPTION
**Pull request by kameronk92**

### CODE REVIEW

**Comment by kameronk92** [comment]
Still need to write the method in class StatTracker to pull info in. Git hub giving me some frustration

File: lib/stats.rb [hunk 1]
```diff
@@ -158,7 +159,112 @@ def season_team_tackles(season)
...
+      end
+    end
+
+    total_goals_against
+  end
+
+  # returns a float for average goals scored by a given team in a given season
+  def average_goals_scored(season_type, season_id, team_id)
+    total_scored = total_goals_scored(season_type, season_id, team_id)
+    total_games = @games_data.count { |game| game[:type] == season_type && game[:season] == season_id && (game[:away_team_id] == team_id || game[:home_team_id] == team_id)}
```
Comment (trpubz) [inline lib/stats.rb:254]: modifying the `#count` filter to this:
```ruby
total_games = @games_data.count { |game|
      game[:type] == season_type && game[:season] == season_id && \
        [game[:away_team_id], game[:home_team_id]].include?(team_id)
    }
``` 
can improve readability and cognitive complexity a few fold.  this revision drops 2 `==` equates. from the logic, this way adds the team ids to an array and then checks for the `team_id`.
Additionally, code lines should have a max length of ~80 and the `\` escape character allows you to move the logic down to the next line without breaking flow.

Comment (trpubz) [reply lib/stats.rb:254]: please use this refactoring and apply it to `#average_goals_against`

Comment (kameronk92) [reply lib/stats.rb]: Updated accordingly, thanks for the tips!

**Review by bkchilidawg — APPROVED** [review]

**Pull request merged**